service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -1,7 +1,6 @@
 package uk.gov.hmcts.reform.fpl.utils;
```

Comment:
Shouldn't this leave with tests?

Reply:
I think so but I think it also used within both integration and unit tests so one breaks if it is placed in the package of the other

Reply:
YES. I despise this class. However, it is used in integration AND unit tests so isn't as simple as just moving. I am not addressing within this PR

Reply:
Leave means live! :D but yes happy to not address it here.